tests/system/feature_store/test_feature_store.py

```diff
@@ -923,6 +923,8 @@ def test_passthrough_feature_set(self, engine):
 
         expected = source.to_dataframe().set_index("patient_id")
 
+        start_time = expected["timestamp"][10]
```

Comment:
@gtopper Why it's a problem?
It the same data always and I execute the filter locally in the test and in the get_offline..

Reply:
Well, it's probably not a problem because
* 10 is just an arbitrary number, that might as well be 9 or 11
* The file is sorted by time

The only issue I see with this is that these facts aren't obvious to anyone reading the code. So a clarification may be in order (maybe a short comment).